benchmark/benchmain/main.go

```diff
@@ -386,20 +388,11 @@ func makeFuncUnary(bf stats.Features) (rpcCallFunc, rpcCleanupFunc) {
 }
 
 func makeFuncStream(bf stats.Features) (rpcCallFunc, rpcCleanupFunc) {
-	clients, cleanup := makeClients(bf)
-
-	streams := make([][]testgrpc.BenchmarkService_StreamingCallClient, bf.Connections)
-	for cn := 0; cn < bf.Connections; cn++ {
-		tc := clients[cn]
-		streams[cn] = make([]testgrpc.BenchmarkService_StreamingCallClient, bf.MaxConcurrentCalls)
-		for pos := 0; pos < bf.MaxConcurrentCalls; pos++ {
+	streams, req, cleanup := setupStream(bf, workloadsStreaming)
```

Comment:
This second argument to setupStream is ignored in this case it's not workloadsUnconstrained. Can we change the argument of the setupStream helper to unconstrained bool, and set it to false here and true where you want unconstrained.